### ISSUE TITLE
FLASHDeconv msAlign Export

### ISSUE THREAD

**Issue by jgpavek** [OpenMS/OpenMS#7613]
It seems like there is a maximum allowed number of deconvoluted masses for the FLASHDeconv msalign output. The masses per spectrum contains 545 deconvoluted masses, while the msalign file only contains 500. Discovered the issue when I had 3 different files deconvoluted and all msalign files contained exactly 500 masses.

**Comment by KyowonJeong** [comment]
Yes. TopFD also has the same limit. This is because msalign was designed to be used within TopPIC search engine, which takes only up to 500 masses.
